app/src/controllers/initialData/sagas.js

```diff
@@ -44,13 +44,13 @@ function* fetchInitialData() {
     const { payload: activeProject } = yield take(SET_ACTIVE_PROJECT);
     yield put(fetchProjectAction(activeProject));
     yield take(FETCH_PROJECT_SUCCESS);
-    yield put(fetchPluginsAction());
     yield put(fetchGlobalIntegrationsAction());
     yield put(authSuccessAction());
   } else {
     yield put(resetTokenAction());
```

Comment:
You need to fetch **public** plugins somewhere here when the https://jiraeu.epam.com/browse/EPMRPP-76240 will be completed by @chingiskhan-epam .